### DISCUSSION TITLE
How do i switch frames using buttons?

### DISCUSSION THREAD

**Discussion by Ryzelz** [TomSchimansky/CustomTkinter#1565] [Q&A]
How do i switch frame while being dynamically add new pages while separated by classes in the future. i used 

`        #This is for spalling alot of frames into one variable
        self.frames = {}
        #dynamicallly add pages without breaking the app
        for F in (StartPage, PageOne, PageTwo):
            
            # Set the frame to the class calling container = parent, self = controller
            frame = F(container, self)
            self.frames[F] = frame
            # grid is superior
            frame.grid(row=0, column = 0, sticky="nsew")`
            
           but its not forwing for some reason.

**Comment by MaxNiark** [comment]
please check my post in dat forum :)
